assignments/lambda-classes.js

```diff
@@ -26,6 +26,10 @@ class Instructor extends Person {
 	grade(student, subject){
 		return `${student.name} receives a perfect score on ${subject}`;
 	}
+
+	gradeReview(student, pointsAdjust){
+		return student.grade = student.grade += Math.random() + pointsAdjust;
+	}
```

Comment:
There's no need to use the assignment operator here with the shorthand addition operator right after. You could just use the shorthand addition operator!
```js
return student.grade += Math.random() + pointsAdjust;
```